status/statusgo_backend_new/stickers.nim

```diff
@@ -1,6 +1,28 @@
 import json
-import ../statusgo_backend_new/eth as eth
+import ./types, ./eth
+import core, utils
+import response_type
 
+proc sendStickerMessage*(
+    chatId: string,
+    replyTo: string,
+    stickerHash: string,
+    stickerPack: int,
+    preferredUsername: string = ""
+    ): RpcResponse[JsonNode] {.raises: [Exception].} =
+  callPrivateRPC("sendChatMessage".prefix, %* [
+    {
+      "chatId": chatId,
+      "text": "Update to latest version to see a nice sticker here!",
+      "responseTo": replyTo,
+      "ensName": preferredUsername,
+      "sticker": {
+        "hash": stickerHash,
+        "pack": stickerPack
+      },
+      "contentType": ContentType.Sticker.int
```

Comment:
@jrainville since both `sendChatMessage` and `sendStickerMessage` actually call the same `sendChatMessage` form the `status-go`, why don't you just make a single `sendChatMessage` here, which will have all possible parameters (union of above mentioned methods) and call it from your service from 2 convenient methods `sendChatMessage` and `sendStickerMessage` with appropriate set of params.

@jrainville please update that and as a part of this change remove added `status/statusgo_backend_new/types.nim` file.

Reply:
Done